src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -246,6 +252,92 @@ internal void UpdatePlacementTarget()
                 stepUIPopup.PlacementTarget = ownedWindow;
                 UpdateLocation();
             }
+            //This case will be used for UIElements that are in the Dynamo VisualTree but they are shown until there is a user interaction (like the SideBar cases)
+            var hostUIElement = Guide.FindChild(MainWindow, HostPopupInfo.HostUIElementString);
+            if (hostUIElement == null)
+                return;
+            HostPopupInfo.HostUIElement = hostUIElement;
+            stepUIPopup.PlacementTarget = hostUIElement;
+            UpdateLocation();
+        }
+
+        /// <summary>
+        /// This method will update the CutOff rectangle size everytime that the step change
+        /// </summary>
+        /// <param name="bVisible">It will say it the CutOff Area will be disabled or enable</param>
```

Comment:
same comment here

Reply:
@QilongTang fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12277/commits/7b9f128012709949425ccbf45f7bf3f52d19c89a